diff --git a/lib/features/home/widget/home_page.dart b/lib/features/home/widget/home_page.dart
index dc43034..5db2689 100644
--- a/lib/features/home/widget/home_page.dart
+++ b/lib/features/home/widget/home_page.dart
@@ -4,6 +4,7 @@ import 'package:gap/gap.dart';
 import 'package:hiddify/core/app_info/app_info_provider.dart';
 import 'package:hiddify/core/localization/translations.dart';
 import 'package:hiddify/core/router/bottom_sheets/bottom_sheets_notifier.dart';
+import 'package:hiddify/core/whitelabel/preset.dart';
 import 'package:hiddify/features/home/widget/connection_button.dart';
 import 'package:hiddify/features/profile/notifier/active_profile_notifier.dart';
 import 'package:hiddify/features/profile/widget/profile_tile.dart';
@@ -68,15 +69,17 @@ class HomePage extends HookConsumerWidget {
               onPressed: () => ref.read(bottomSheetsNotifierProvider.notifier).showQuickSettings(),
             ),
           ),
-          const Gap(8),
-          Semantics(
-            key: const ValueKey("profile_add_button"),
-            label: t.pages.profiles.add,
-            child: IconButton(
-              icon: Icon(Icons.add_rounded, color: theme.colorScheme.primary),
-              onPressed: () => ref.read(bottomSheetsNotifierProvider.notifier).showAddProfile(),
+          if (!WhitelabelConfig.hideProfileManagement) ...[
+            const Gap(8),
+            Semantics(
+              key: const ValueKey("profile_add_button"),
+              label: t.pages.profiles.add,
+              child: IconButton(
+                icon: Icon(Icons.add_rounded, color: theme.colorScheme.primary),
+                onPressed: () => ref.read(bottomSheetsNotifierProvider.notifier).showAddProfile(),
+              ),
             ),
-          ),
+          ],
           const Gap(8),
         ],
       ),
